dataframe/dataframe.go

```diff
@@ -887,15 +898,23 @@ func LoadRecords(records [][]string, options ...LoadOption) DataFrame {
 	if cfg.hasHeader && len(records) <= 1 {
 		return DataFrame{Err: fmt.Errorf("load records: empty DataFrame")}
 	}
+	if len(cfg.names) > len(records[0]) {
```

Comment:
Mmmm, yeah for now I prefer to switch it to my suggestion, if we receive feedback of enough people that they prefer Pandas system we can revert to it.